containers/multisite/wp-config.php

```diff
@@ -93,14 +94,13 @@
 }
 
 /* Multisite */
-
-// define('WP_ALLOW_MULTISITE', true);
-define('MULTISITE', true);
+define('WP_ALLOW_MULTISITE', true);
```

Comment:
@ktbartholomew When I run docker-compose down --rmi all && docker-compose up -d, I am faced with "Error establishing a database connection" and cannot access the site whatsoever. When using WP_ALLOW_MULTISITE, I can access Wordpress, and navigate to /wp-admin/network.php which then prompts me to install multisite.

Perhaps you can advise on this?